src/benchmarks/gc/src/analysis/managed-lib/Analysis.cs

```diff
@@ -505,18 +505,20 @@ public class TracedProcesses
 
         public static void PrintEventsWithoutTraceLog(string tracePath, Func<TraceEvent, bool> filter, uint? maxEvents)
         {
-            using TraceEventDispatcher source = getSource(tracePath);
+            using TraceEventDispatcher source = TraceEventDispatcher.GetDispatcherFromFileName(tracePath);
```

Comment:
>using TraceEventDispatcher source = TraceEventDispatcher.GetDispatcherFromFileName(tracePath); [](start = 12, length = 94)

you could do
`using TraceEventDispatcher source = TraceEventDispatcher.GetDispatcherFromFileName(tracePath)!;`

that way you don't need to do source! at all the other places.